examples/secure_token_storage.py

```diff
@@ -7,9 +7,11 @@
 # There are other ways to load environment variables but this is one
 from dotenv import load_dotenv # https://pypi.org/project/python-dotenv/
 
-# load_dotenv reads from a file called .env in the same directory as the python files which should roughly look like TOKEN="1234567890"
+# load_dotenv reads from a file called .env in the same directory as the python files which should roughly look like BOT_TOKEN="1234567890"
 load_dotenv()
-token = os.getenv('BOT_TOKEN')
+
+token = os.getenv('BOT_TOKEN') #getting the token
+
 bot = commands.Bot()
```

Comment:
Maybe add a custom prefix since it's currently not optional?
```suggestion
bot = commands.Bot(command_prefix="!")
```

Reply:
Lets keep the consistency across the examples and use "?"

Reply:
Yes, I messed it up.

Reply:
Yea I'll do it tomorrow.... I'm on my phone rn and I need to sleep lol

Reply:
Made the changes... There anything else?